records_mover/db/mysql/mysql_db_driver.py

```diff
@@ -17,11 +17,16 @@
 
 class MySQLDBDriver(DBDriver):
     def __init__(self,
-                 db: Union[sqlalchemy.engine.Engine, sqlalchemy.engine.Connection],
                  url_resolver: UrlResolver,
+                 db: Optional[Union[sqlalchemy.engine.Engine, sqlalchemy.engine.Connection]] = None,
+                 db_conn: Optional[sqlalchemy.engine.Connection] = None,
+                 db_engine: Optional[sqlalchemy.engine.Engine] = None,
                  **kwargs) -> None:
-        super().__init__(db)
+        self.conn_opened_here = False
```

Comment:
why does this class set `conn_opened_here` but `bigquery_db_driver` does not?

Reply:
oversight. The conn_opened_here gets set in the super().__init__

Reply:
resolved!